docs/samples/v1beta1/torchserve/v1/bloom/README.md

```diff
@@ -0,0 +1,197 @@
+# TorchServe example with Huggingface BLOOM model
+In this example we will show how to serve [Large Huggingface models with TorchServe](https://github.com/pytorch/serve/tree/master/examples/Huggingface_Largemodels)
+on KServe.
+
+## Model archive file creation
+
+Clone [pytorch/serve](https://github.com/pytorch/serve) repository,
+navigate to `examples/Huggingface_Largemodels` and follow the steps for creating the MAR file including serialized model and other dependent files.
+
+The above Torchserve example works on shard version on Huggingface models.
+
+The For sharding Huggingface models use the below script, and continue with [this step](https://github.com/pytorch/serve/tree/master/examples/Huggingface_Largemodels#step-2-compress-downloaded-model)
+
+```python
+from transformers import AutoModelForCausalLM, AutoTokenizer
+model_name="bigscience/bloomz-7b1"
+model = AutoModelForCausalLM.from_pretrained(model_name)
+tokenizer = AutoTokenizer.from_pretrained(model_name)
+model.save_pretrained("model/"+model_name, max_shard_size="5GB")
+tokenizer.save_pretrained("model/"+model_name)
+```
+
+TorchServe supports both eager model and torchscript and here we save as the pretrained model. 
+ 
+```bash
+torch-model-archiver --model-name BLOOMSeqClassification --version 1.0 \
+--serialized-file Transformer_model/pytorch_model.bin \
```

Comment:
This is not the sharded model right?

Reply:
That is step is redundant. The Torchserve repo link cover MAR file generation step. I will remove it here.